components/ui/Footer.js

```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import Link from 'next/link'
+import { StyledLinkText, Body1 } from './Typography'
+import theme from '../../util/theme'
+
+const FooterContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 8rem;
+
+  background: ${props => props.backgroundColor}
+    ${props => props.theme.halftones.md};
+`
+
+const FooterLinksContainer = styled.div`
+  display: flex;
+  font-size: 2rem;
+  line-height: 2rem;
+
+  > :not([hidden]) ~ :not([hidden]) {
```

Comment:
I wouldn’t expect hidden to play a role here, how about: ` > *:nth-child(n+2) { border-left-.... }` or `:not(:first-child)` could also work.

Reply:
good call. I copied from Tailwind CSS's `divide-x` source!